api/Dockerfile

```diff
@@ -4,7 +4,7 @@ EXPOSE 5000/tcp
 
 RUN apk update && \
     apk add --virtual build-deps gcc git python3-dev musl-dev && \
-    apk add postgresql-dev
+    apk add postgresql-dev postgresql-client
```

Comment:
This is required to run `wait-for-postgres.sh`, which connects to postgres using this library to verify that it's up and running.